src/commands/flinkStatements.ts

```diff
@@ -245,7 +180,10 @@ export async function submitFlinkStatementCommand(
 
     // Will resolve when the statement is in a viewable state and
     // the results viewer is open.
-    await waitAndShowResults(newStatement);
+    await statementsView.withProgress(`Submitting statement ${newStatement.name}`, async () => {
```

Comment:
Abandon micro-utility waitAndShowResults() entirely in favor of using the statements view's `withProgress()` throbber instead.